UnitsNet/UnitDefinitions/RotationalSpeed.json

```diff
@@ -1,8 +1,11 @@
 ﻿{
-    "Name": "RotationalSpeed",
-    "BaseUnit": "RadianPerSecond",
-    "XmlDoc": "Rotational speed (sometimes called speed of revolution) is the number of complete rotations, revolutions, cycles, or turns per time unit. Rotational speed is a cyclic frequency, measured in radians per second or in hertz in the SI System by scientists, or in revolutions per minute (rpm or min-1) or revolutions per second in everyday life. The symbol for rotational speed is ω (the Greek lowercase letter \"omega\").",
-    "Units": [
+  "Name": "RotationalSpeed",
+  "BaseUnit": "RadianPerSecond",
+  "XmlDoc": "Rotational speed (sometimes called speed of revolution) is the number of complete rotations, revolutions, cycles, or turns per time unit. Rotational speed is a cyclic frequency, measured in radians per second or in hertz in the SI System by scientists, or in revolutions per minute (rpm or min-1) or revolutions per second in everyday life. The symbol for rotational speed is ω (the Greek lowercase letter \"omega\").",
+  "BaseDimensions": {
+    "T": -1
```

Comment:
I just realized something that might block automatically generating conversions between quantities.

Wolfram lists the dimensions like this: `[time]^(-2) [angle]`
http://www.wolframalpha.com/input/?i=radian+per+second+squared

Without angle, which is not a SI base unit, we don't represent that this is a rotational quantity so there is now ambiguity whether `Length / Time` should equal `Speed` or `RotationalSpeed`. Same for [illuminance](http://www.wolframalpha.com/input/?i=illuminance&assumption=%7B%22FS%22%7D+-%3E+%7B%7B%22IlluminanceFromLuminousIntensity%22,+%22I%22%7D%7D&assumption=%7B%22F%22,+%22IlluminanceFromLuminousIntensity%22,+%22r%22%7D+-%3E%221+m%22&assumption=%7B%22F%22,+%22IlluminanceFromLuminousIntensity%22,+%22Enu%22%7D+-%3E%221+lx%22&assumption=%7B%22C%22,+%22illuminance%22%7D+-%3E+%7B%22PhysicalQuantity%22%7D) having `[solid angle]` as a dimension.

The same goes for all the rotational vs linear quantities really.

Am I missing something?

Reply:
Some can be the same. Take a look at PressureChangeRate and PowerDensity for example.

I am not entirely sure how to resolve this just yet. Pa/s is technically valid as both a power density as well as a pressure change rate, but only used in practice as a pressure change rate :)